src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -394,19 +399,15 @@ public static IEnumerable<object[]> ASin_Advanced_TestData()
             yield return new object[] { 0.0, double.MaxValue, 0.0, Math.Log(2.0) + Math.Log(double.MaxValue) };
             yield return new object[] { double.MaxValue, double.MaxValue, Math.PI / 4.0, Math.Log(2.0 * Math.Sqrt(2.0)) + Math.Log(double.MaxValue) };
 
-            // Invalid values
```

Comment:
Ditto for this: preserve in a new "Legacy" test case which only runs on .NET Framework.